publications-2018.bib

```diff
@@ -1,5 +1,60 @@
 % Encoding: US-ASCII
 
+
+@article{Jodlbauer2018,
+  doi = {10.1002/nme.5970},
+  url = {https://doi.org/10.1002/nme.5970},
+  year = {2018},
+  month = oct,
+  publisher = {Wiley},
+  volume = {117},
+  number = {6},
+  pages = {623--643},
+  author = {D. Jodlbauer and U. Langer and T. Wick},
+  title = {Parallel block-preconditioned monolithic solvers for fluid-structure interaction problems},
+  journal = {International Journal for Numerical Methods in Engineering}
+}
+
+@article{Fu2018,
+  doi = {10.1093/imanum/dry001},
+  url = {https://doi.org/10.1093/imanum/dry001},
+  year = {2018},
+  month = mar,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {39},
+  number = {2},
+  pages = {957--982},
+  author = {Guosheng Fu and Yanyi Jin and Weifeng Qiu},
+  title = {Parameter-free superconvergent H(div)-conforming {HDG} methods for the Brinkman equations},
+  journal = {{IMA} Journal of Numerical Analysis}
+}
+
+@article{GonzlezValverde2018,
+  doi = {10.1007/s40571-018-0199-2},
+  url = {https://doi.org/10.1007/s40571-018-0199-2},
+  year = {2018},
+  month = jun,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {6},
+  number = {1},
+  pages = {85--96},
+  author = {Ismael Gonz{\'{a}}lez-Valverde and Jos{\'{e}} Manuel Garc{\'{\i}}a-Aznar},
+  title = {An agent-based and {FE} approach to simulate cell jamming and collective motion in epithelial layers},
+  journal = {Computational Particle Mechanics}
+}
+
+@incollection{Guermond2018,
+  doi = {10.1007/978-3-319-78325-3_14},
+  url = {https://doi.org/10.1007/978-3-319-78325-3_14},
+  year = {2018},
+  month = jul,
+  publisher = {Springer International Publishing},
+  pages = {251--272},
+  author = {Jean-Luc Guermond and Bojan Popov and Laura Saavedra and Yong Yang},
+  title = {Arbitrary Lagrangian-Eulerian Finite Element Method Preserving Convex Invariants of Hyperbolic Systems},
+  booktitle = {Computational Methods in Applied Sciences}
+}
```

Comment:
same here. book seems to be published 2019 https://link.springer.com/book/10.1007/978-3-319-78325-3